### PR TITLE
styles(input): [input] add input types

### PR DESCRIPTION
**Pull request by jxhhdx**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by kagol — APPROVED** [review]

File: packages/renderless/src/input/index.ts [hunk 1]
```diff
@@ -226,7 +227,7 @@ export const handleInput =
       return
     }
 
-    if ((event.target as HTMLInputElement | HTMLTextAreaElement).value === state.nativeInputValue) {
+    if ((event.target as HTMLInputElement | HTMLTextAreaElement).value === unref(state.nativeInputValue)) {
```
Comment (kagol) [inline packages/renderless/src/input/index.ts:230]: @jxhhdx 
我看很多地方包裹unref，这样最组件逻辑是否有影响呢？

Comment (yoyo201626) [reply packages/renderless/src/input/index.ts:230]: 加个unref是要做什么呢？没必要吧，我理解state.nativeInputValue 拿到的就是unref

File: packages/renderless/src/input/index.ts [hunk 2]
```diff
@@ -280,14 +281,14 @@ export const setNativeInputValue =
       return
     }
 
-    if (input.value === state.nativeInputValue) {
+    if (input.value === unref(state.nativeInputValue)) {
```
Comment (yoyo201626) [inline packages/renderless/src/input/index.ts:284]: 这里也是加个unref是要做什么呢？没必要吧

**Comment by kagol** [comment]
冲突和检视意见长时间未解决

**Pull request closed without merging**